Mail2Bug/WorkItemManagement/TFSWorkItemManager.cs

```diff
@@ -304,6 +304,35 @@ public void ModifyWorkItem(int workItemId, string comment, Dictionary<string, st
             workItem.Save();
         }
 
+        public SimpleWorkItem GetWorkItem(int workItemId)
+        {
+            if (workItemId <= 0) return null;
+            var tfsWorkItem = _tfsStore.GetWorkItem(workItemId);
+
+            var workItem = new SimpleWorkItem()
+            {
+                Id = tfsWorkItem.Id,
+                Title = tfsWorkItem.Title,
+                AssignedTo = tfsWorkItem.Fields["Assigned To"].Value as string,
+                State = tfsWorkItem.State,
+                Fields = AsDictionary(tfsWorkItem.Fields)
+            };
+
+            return workItem;
+        }
+
+        private Dictionary<string, string> AsDictionary(FieldCollection fields)
```

Comment:
Done. Replaced the dictionary with classes IWorkItemFields and TfsWorkItemFields, to wrap the object and provide the values on-demand.